config/jobs/gardener/gardener-pull-tests.yaml

```diff
@@ -45,6 +53,69 @@ presubmits:
         args:
         - test-integration
         resources:
+          limits:
+            memory: 4000Mi
           requests:
-            cpu: 6
+            cpu: 4
             memory: 2000Mi
+periodics:
```

Comment:
Maybe the file should be renamed to `gardener-tests.yaml`? Now it doesn't contain just PR jobs anymore.
Otherwise, you could split it up into `gardener-unit.yaml` and `gardener-integration.yaml` and include the corresponding presubmit and periodic.